app/build.gradle

```diff
@@ -123,7 +122,7 @@ dependencies {
     implementation 'com.google.android.material:material:1.0.0'
     implementation 'androidx.lifecycle:lifecycle-extensions:2.1.0'
     implementation 'androidx.lifecycle:lifecycle-viewmodel-ktx:2.1.0'
-    implementation 'androidx.lifecycle:lifecycle-livedata-ktx:2.2.0-rc03'
+    implementation 'androidx.lifecycle:lifecycle-livedata-ktx:2.3.0-alpha05'
```

Comment:
Move this to it's own commit

Reply:
Also add a comment at the top here (around dependencies {) to make sure to update libs in the libs dir / update libs/Android.bp or so on.

Reply:
Maybe also add a note about how to retrieve the prebuild aars, either from local gradle cache or maven repo. This will be handy when we want to bump this alpha version to something more stable.

Reply:
[2.3.0-alpha06](https://developer.android.com/jetpack/androidx/releases/lifecycle#2.3.0-alpha06) is out with some bugfixes, maybe we should move to that directly?